src/Output/LogOutputBuffer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Output;
+
+final readonly class LogOutputBuffer implements OutputBuffer
+{
+    public function __construct(private string $path)
+    {
+    }
+
+    public function clear(): self
+    {
+        file_put_contents($this->path, '');
+
+        return $this;
+    }
+
+    public function read(): string
+    {
+        return file_get_contents($this->path);
+    }
+
+    public function write(string $contents): void
+    {
+        $contents = '[' . date('Y-m-d H:i:s') . '] ' . $contents;
```

Comment:
We may want to think about creating a first-party wrapper around date so we can test this badboy easily too

Reply:
It would allow to handle timezones.